image-has-acc-name: applicability should not use the acc tree (#1381)
act-rules/act-rules.github.io: _rules/image-non-empty-accessible-name-23a2a8.md

  - CSS Styling
acknowledgments:
  authors:
    - Wilco Fiers

>   previous_authors:

Author: I thought this was necessary because of the applicability change, would like a yay or nay on that.

Reviewer: Yay!

Reviewer 2: No real opinion on that.
Maybe we should have some clear guidelines on the topic at some point 😄

## Applicability

> The rule applies to HTML `img` elements and HTML elements with the [semantic role][] of `img`, for which all the following is true about it and all its [ancestors][] in the [flat tree][]:

Reviewer 3: ```suggestion
The rule applies to HTML `img` elements and HTML elements with the [semantic role][] of `img`, for which all the following is true about them and all their [ancestors][] in the [flat tree][]:
```

- the element does not have an `aria-hidden` [attribute value][] of "true"; and
- the element does not have a [computed][] [display][] property of "none"; and

> - the element does not have a [computed][] [visibility][] property of "hidden".

Author: Did it slightly different, but good idea. I don't think the example descriptions need to change though.

#### Inapplicable Example 3

This `img` element is hidden with `aria-hidden` set to "true".

```html
<img src="/test-assets/shared/w3c-logo.png" aria-hidden="true" />
```

#### Inapplicable Example 4

This `img` element is hidden because its parent has `display: none`.

```html
<div style="display: none">
	<img src="/test-assets/shared/w3c-logo.png" />
</div>
```

#### Inapplicable Example 5

> This `img` element is hidden with `visibility: hidden`.

Reviewer 2: ```suggestion
This `img` element is hidden because its parent has `visibility: hidden`.
```

Author: I think that's an unnecessary detail that is obvious from the example itself.

> <div aria-label="W3C logo"></div>

Reviewer 4: Would it be worth adding the `background-image` property for consistency?

Author: Yaknow what, I don't think this is a good example. Putting aria-label on a random `div` does very little. I'm going to take this out.
